internal/utils/utils.go

```diff
@@ -20,52 +20,17 @@ package utils
 
 import (
 	"context"
-	"fmt"
-	"os"
+	"github.com/submariner-io/subctl/internal/constants"
+	"github.com/submariner-io/subctl/internal/exit"
 
 	"github.com/pkg/errors"
-	"github.com/submariner-io/subctl/pkg/subctl/operator/submarinercr"
-	"github.com/submariner-io/subctl/pkg/version"
 	"github.com/submariner-io/submariner-operator/api/submariner/v1alpha1"
 	subOperatorClientset "github.com/submariner-io/submariner-operator/pkg/client/clientset/versioned"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	v1opts "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/rest"
 )
 
-// PanicOnError will print the subctl version and then panic in case of an actual error.
-func PanicOnError(err error) {
-	if err != nil {
-		fmt.Fprintln(os.Stderr, "")
-		version.PrintSubctlVersion(os.Stderr)
-		fmt.Fprintln(os.Stderr, "")
-		panic(err.Error())
-	}
-}
-
-// ExitOnError will print your error nicely and exit in case of error.
-func ExitOnError(message string, err error) {
-	if err != nil {
-		ExitWithErrorMsg(fmt.Sprintf("%s: %s", message, err))
-	}
-}
-
-// ExitWithErrorMsg will print the message and quit the program with an error code.
-func ExitWithErrorMsg(message string) {
-	fmt.Fprintln(os.Stderr, message)
-	fmt.Fprintln(os.Stderr, "")
-	version.PrintSubctlVersion(os.Stderr)
-	fmt.Fprintln(os.Stderr, "")
-	os.Exit(1)
-}
-
-// ExpectFlag exits with an error if the flag value is empty.
-func ExpectFlag(flag, value string) {
-	if value == "" {
-		ExitWithErrorMsg(fmt.Sprintf("You must specify the %v flag", flag))
-	}
-}
-
 func GetSubmarinerResourceWithError(config *rest.Config) (*v1alpha1.Submariner, error) {
```

Comment:
I think there was another function to get `submariner`. Can't find it. @tpantelis Do you remember it?

Reply:
AFAICT, this function is now only used by `restconfig.CheckVersionMismatch` so we can simply inline the code there - it's  not much. Plus, as you probably know, I'm not a fan of `utils` packages 😄 

```Go
func (rcp *Producer) CheckVersionMismatch(cmd *cobra.Command, args []string) error {
	...
	operatorClient, err := subOperatorClientset.NewForConfig(config.Config)
	exit.OnErrorWithMessage(err, "Error creating operator clientset")

	submariner, err := operatorClient.SubmarinerV1alpha1().Submariners(constants.OperatorNamespace).Get(
		context.TODO(), constants.SubmarinerName, v1.GetOptions{})
	if apierrors.IsNotFound(err) {
		return nil
	}

	exit.OnErrorWithMessage(err, "Error obtaining Submariner resource")

	if submariner.Spec.Version != "" {
		...
	}

	return nil
}
```